fzf-tab.zsh

```diff
@@ -340,6 +340,11 @@ build-fzf-tab-module() {
   pushd $FZF_TAB_HOME/modules
   CPPFLAGS=-I/usr/local/include CFLAGS="-g -Wall -O2" LDFLAGS=-L/usr/local/lib ./configure --disable-gdbm --without-tcsetpgrp ${MACOS:+DL_EXT=bundle}
   make -j$(nproc)
+  if [[ $? -ne 0 ]]; then
+	  echo "The module building has failed. See the output above for details." >&2
```

Comment:
I think we can make it more clearly here by printing colored text, like `print -P "%F{red}fail%f %F{green}success%f"`

Reply:
Good idea!